tests/scripts/list-identifiers.sh

```diff
@@ -68,7 +68,7 @@ fi > _identifiers
 
 if [ $( wc -l < _identifiers ) -eq $( wc -l < _decls ) ]; then
     rm _decls
-    egrep -v '^(u?int(16|32|64)_t)$' _identifiers | sort > identifiers
+    egrep -v '^(u?int(16|32|64)_t|MBEDTLS_CHECK_RETURN_(CRITICAL|OPTIONAL|TYPICAL))$' _identifiers | sort > identifiers
```

Comment:
Note: this will conflict with https://github.com/ARMmbed/mbedtls/pull/1638 - if you don't mind I'd like to get 1638 merged first and then you can quickly resolve the conflict here, as the other way round would likely result in higher latency.